css-grid-1/grid-items/grid-item-containing-block-001.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>CSS Grid Layout Test: Grid item sizing</title>
+        <link rel="author" title="Tomek Wytrebowicz" href="mailto:tomalecpub@gmail.com">
+        <link rel="help" href="http://www.w3.org/TR/css-grid-1/#grid-items" title="4. Grid Items">
+        <meta name="assert" content="A grid item is sized within the containing block defined by its grid area">
+        <link rel="match" href="../../css21/reference/ref-filled-green-100px-square.xht">
+        <style type="text/css">
```

Comment:
Also you don't need `<html>` and `<head>` tags. See the [template](http://testthewebforward.org/docs/test-templates.html).

More info about optional tags: http://www.w3.org/TR/2011/WD-html5-20110525/syntax.html#optional-tags